command/polybftsecrets/params_test.go

```diff
@@ -59,3 +61,31 @@ func fileExists(filename string) bool {
 
 	return !info.IsDir()
 }
+
+func Test_getResult(t *testing.T) {
+	// Creates test directory
+	dir, err := os.MkdirTemp("", "test")
+	defer os.RemoveAll(dir)
+
+	sm, err := helper.SetupLocalSecretsManager(dir)
+	require.NoError(t, err)
+
+	ip := &initParams{
+		generatesAccount: true,
+		generatesNetwork: true,
+		chainID:          1,
+	}
+
+	_, err = ip.initKeys(sm)
+	require.NoError(t, err)
+
+	res, err := ip.getResult(sm, []string{})
+	require.NoError(t, err)
+
```

Comment:
Could we add some assertions to check `BLSPubkey`, `Address` etc. as well?

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1248/commits/1f91d332597e272574c10bd34c06538e81735188